packages/office-ui-fabric-react/src/components/Dialog/examples/Dialog.Basic.Example.tsx

```diff
@@ -31,6 +31,7 @@ export class DialogBasicExample extends React.Component<{}, IDialogBasicExampleS
         <Dialog
           hidden={this.state.hideDialog}
           onDismiss={this._closeDialog}
+          className="XXXXXXXXX"
```

Comment:
If you want to do this to generate new snapshot tests for verification I suggest using a less jarring `className` value such as `"foo"`

Reply:
Though I think since we're not applying any styles, we should simply move this to be encapsulated in a unit test which verifies a `className` on `Dialog` is only applied once.

If we add it to an example and generate snapshots that is a valid way of testing. However, developers frequently copy our examples to bootstrap their development. Having an artifact like an empty `className` would simply be more cognitive overload for them to deal with.

Reply:
It should be a straight-forward unit test to add...